client/asset/btc/spv_wrapper.go

```diff
@@ -1711,7 +1726,7 @@ func (w *spvWallet) getWalletTransaction(txHash *chainhash.Hash) (*GetTransactio
 		TimeReceived: uint64(details.Received.Unix()),
 	}
 
-	if details.Block.Height != -1 {
+	if details.Block.Height > 0 {
```

Comment:
Does not block height == 0 mean tx is in the genesis block and shouldn't we set the block details if so?

Reply:
Yes, that's my understanding too, it shouldn't be an issue in practice because currently SPV wallets are dexc-managed (so there no way such a wallet could have BTC transactions in genesis block); that could change in the future though, so maybe better change this now too, **done**.